.github/pull_request_template.md

```diff
@@ -1,69 +1,74 @@
-<!---
-Please do not delete this text completely, but read the text below and keep
-items that seem relevant.
-If in doubt, just keep everything and add your own text at the top, a reviewer
-will update the checklist for you.
+<!--
```

Comment:
I wanted to avoid having to delete a bunch of text to make place for your own text. In my opinion, the PR template should be ready to fill in, so I moved all the stuff we should delete anyway to a comment. We already have a bunch of comments in the current/previous template anyway, and I don't think this lead to any problems 😅